frontend/src/components/shared/RepoTabs.vue

```diff
@@ -10,14 +10,16 @@
         :userName="userName"
         :namespacePath="path"
         :repo="repoDetail"
-        :enableEndpoint="repoDetail.enableInference"
-        :enableFinetune="repoDetail.enableFinetune"
-        :enableEvaluation="repoDetail.enableEvaluation"
+        :enableEndpoint="repoDetail.enable_inference"
```

Comment:
Here I think it should be 'repoDetail.enableInference', b/c repoDetail is actually from repoDetailStore

Reply:
fixed